### PR TITLE
Updates for new T&S Telemetry in ts_xml 5.0

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: start_docker_dev.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 HSUSER=headerservice
-TAGNAME=4.0.0-5.0.0-salobj_5.8.0-aths_2.0.0
+TAGNAME=4.1.0-5.0.0-salobj_5.10.0-aths_2.0.0
```
Comment (mareuter) [inline start_docker_dev.sh]: Is the ATHS tag version updated just before a new release is done?

File: start_docker_service.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-TAGNAME=4.0.0-5.0.0-salobj_5.8.0-aths_2.0.0
+TAGNAME=4.1.0-5.0.0-salobj_5.10.0-aths_2.0.0
```
Comment (mareuter) [inline start_docker_service.sh]: Ditto as above.

File: start_docker_dev.sh (hunk 1 above)
Comment (menanteau) [reply start_docker_dev.sh]: Yes, usually after the PR is done. But I should change `aths` to just `hs` or `headerservice`

**Comment by menanteau** [comment]
@mareuter I've update the future version as 2.0.1, which I will tag after I merge to master.
Thanks

**Review by mareuter — APPROVED** [review]
Looks good to me. Thanks for the updates.

**Pull request merged**